### PR TITLE
Chore: Remove the rest of unnecessary files from package

### PR DESCRIPTION
**Pull request by andresmgot**
**What is this feature?**

Follow up of #73609. Rather than just omitting the `pkg` folder, remove everything that's not the `src` folder which includes the same files than in `main`.

### CODE REVIEW

File: pkg/build/packaging/grafana.go [hunk 1]
```diff
@@ -444,11 +444,15 @@ func copyPubDir(grafanaDir, tmpDir string) error {
...
 
 	// Remove plugin backend code.
-	pluBeFiles, err := filepath.Glob(filepath.Join(tgtPubDir, "plugins", "*", "pkg"))
+	pluginFiles, err := filepath.Glob(filepath.Join(tgtPubDir, "plugins", "*", "*"))
 	if err != nil {
 		return err
 	}
-	for _, p := range pluBeFiles {
+	for _, p := range pluginFiles {
+		if filepath.Base(p) == "src" {
```
Comment (marefr) [inline pkg/build/packaging/grafana.go:452]: Not sure this works with sub-directories of src directories?

**Review by marefr — APPROVED** [review]
LGTM with a concern

File: pkg/build/packaging/grafana.go (hunk 1 above)
Comment (andresmgot) [reply pkg/build/packaging/grafana.go:452]: The `Glob` pattern only checks the main directory of the plugin, it doesn't go deeper than that:

image

Comment (marefr) [reply pkg/build/packaging/grafana.go:452]: Nice then you're good to go

**Pull request merged**